nodes/logic/custom_switcher.py

```diff
@@ -102,8 +109,17 @@ def process(self):
         else:
             self.string_values.clear()
 
+        self.ensure_user_list_sync()
         self.outputs['Item'].sv_set([[i for i, b in enumerate(self.user_list) if b]])
```

Comment:
Looks like fixing the string is enough. And in this cases selection still will be active when number of input elements increases.
```suggestion
        self.outputs['Item'].sv_set([[i for i, b in enumerate(self.user_list[:self.string_values]) if b]])
```